include/map.h

```diff
@@ -2,14 +2,14 @@
  * @file map.h
  * @brief decleration of the map
  *
- * Function decleration and type definition for the map.
+ * Function decleration and type definition for the game's map.
  */
 #ifndef MAP_H_
 #define MAP_H_
 #ifdef _WIN32
-#include <SDL.h>
-#include <sdl_ttf.h>
-#include <SDL_image.h>
+#include <SDL2/SDL.h>
+#include <SDL2/sdl_ttf.h>
+#include <SDL2/SDL_image.h>
```

Comment:
I think we are not using sdl_image anymore. we can delete this line from here and also from makefile.